go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/spiffe/spire
 
-go 1.24.6
```

Comment:
We should update to 1.24.8 to prevent jumping to the next minor release in a patch release.